src/skvhttpclient/src/skvhttp/client/SKVClient.cpp

```diff
@@ -101,7 +101,8 @@ boost::future<Response<>> TxnHandle::write(dto::SKVRecord& record, bool erase, d
         .isDelete = erase,
         .precondition = precondition,
         .value = record.storage.share(),
-        .fieldsForPartialUpdate = std::vector<uint32_t>{}
+        .fieldsForPartialUpdate = std::vector<String>{},
+        .key = dto::SKVRecord::Storage{}
```

Comment:
You can remove the key from here so that we don't give the user an option to specify a separate key and value record as mentioned above.

Reply:
Done.